app/app.go

```diff
@@ -130,6 +130,7 @@ import (
 )
 
 const appName = "Finschia"
+const upgradeName = "v2-Daisy"
```

Comment:
Where can I find a list of which names will be used?

Reply:
Currently, the upcoming name is `Daisy`. We should expose the upgrade plans to the public. Thank you for asking.